### ISSUE TITLE
Add Translations for Incomplete Web UI Sections

### ISSUE THREAD

**Issue by ironAiken2** [lablup/backend.ai-webui#4300]
![image-20250724-083631.png](image-20250724-083631.png){width=662 height=194}

We have identified several sections of the Web UI that are not fully translated. To improve user experience and accessibility, we need to add the missing translations.

JIRA Issue: FR-1286
